src/parser.rs

```diff
@@ -1790,6 +1791,13 @@ impl<'a> Parser<'a> {
         })
     }
 
+    pub fn parse_explain(&mut self) -> Result<Statement, ParserError> {
+        let analyze = self.parse_keyword(Keyword::ANALYZE);
+        let query = Box::new(self.parse_query()?);
```

Comment:
It looks like most databases support a statement in explain (analyze) instead of "only" a query?

Reply:
Goot catch! Thanks